lib/services/apigateway/index.js

```diff
@@ -83,6 +83,16 @@ function getCompiledApiGatewayTemplate(stackName, ownServiceContext, dependencie
         tags: deployPhaseCommon.getTags(ownServiceContext)
     }
 
+		// Add binary media types if specified
+		if(serviceParams.binary_media_types)
+    {
+      handlebarsParams.binaryMediaTypes=[];
+      for(let type of serviceParams.binary_media_types)
```

Comment:
Wait I think I totally missed line 87, sorry about that, that's what I was looking for!